### PR TITLE
media: Implement utc_media_player

### PR DESCRIPTION
**Pull request by junmin-kim**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [21500466336b492554082dfa8ec9537358105396] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [21500466336b492554082dfa8ec9537358105396] - Code Rule Check OK.

File: framework/src/media/FileInputDataSource.cpp [hunk 1]
```diff
@@ -25,13 +25,15 @@ namespace stream {
 
 FileInputDataSource::FileInputDataSource()
 	: InputDataSource()
-	, mDataPath("")
-	, mFp(nullptr)
+	, mDataPath{""}
```
Comment (jsdosa) [inline framework/src/media/FileInputDataSource.cpp]: why change that~?

Comment (junmin-kim) [reply framework/src/media/FileInputDataSource.cpp]: I revert it.

**Comment by seinfra** [comment]
Target : [95f26ef8e2b70a44690384ca244d0c8e267343e3] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [95f26ef8e2b70a44690384ca244d0c8e267343e3] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a04c251f833c119c29be44a20b6a18258616bd04] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a04c251f833c119c29be44a20b6a18258616bd04] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [797430288155f778e305cc82a2250440f14e1039] -

**Comment by seinfra** [comment]
Target : [9f9873c1077ba014be2349456aca1b43fab6c0f4] -

**Comment by seinfra** [comment]
Target : [dab4366feda9e61a818bb58a3936990f6dafaebe] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [dab4366feda9e61a818bb58a3936990f6dafaebe] - Code Rule Check OK.

File: apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp [hunk 2]
```diff
@@ -54,7 +54,7 @@ TEST_F(SimpleMediaPlayerTest, getState)
 
 TEST_F(SimpleMediaPlayerTest, create)
 {
-	ASSERT_EQ(mp->create(), media::PLAYER_OK);
+	EXPECT_EQ(mp->create(), media::PLAYER_OK);
 	EXPECT_EQ(mp->getState(), media::PLAYER_STATE_IDLE);
```
Comment (jsdosa) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp]: please , remove getstate() function.
When the user and our framework access that function, the state is incorrect.

Comment (junmin-kim) [reply apps/examples/testcase/ta_tc/media/utc/utc_media_player.cpp]: I'll change in another PR

**Pull request merged**